### ISSUE TITLE
CyanideAndHappiness download fails

### ISSUE THREAD

**Issue by simonszu** [webcomics/dosage#55]
Hey guys, 

i have made a reinstall of dosage on a fresh host, and i noticed that several webcomics downloads fail because of an outdated regex or something. I think i will just create an issue for every comic, ok?

First: CyanideAndHappiness

     ERROR: Patterns ['<\\s*[aA]\\s+[^>]*prev[^>]*\\s+[hH][rR][eE][fF]\\s*=\\s*"(/comics/\\d+/)"[^>]*[^>]*>'] not found at URL http://www.explosm.net/comics/.

**Comment by jschpp** [comment]
I can't reproduce any of them. 
Steps I took:
```
git clone https://github.com/webcomics/dosage.git
pip install pbr
pip install six
pip install requests
pip install colorama
pip install lxml
pip install cssselect
cd dosage
./dosage CyanideAndHappiness Dilbert FonFlatter SandraAndWooGerman xkcd AhoiPolloi GaiaGerman LoadingArtist SMBC LookingForGroup Ruthe DieFruehreifen JohnnyWander PennyArcade SafelyEndangered Wumo Arcamax/ArcticCircle
```
[Output](https://gist.github.com/jschpp/c60209b8851a7cd7e26e99291c3f7eb8)
I ran in to 2 erros namely:
```
SMBC> ERROR: XPath //img[@id="comic"]/@title did not match anything at URL http://www.smbc-comics.com/.
Wumo> ERROR: XPath //article[1]//div[@class="box-content"]//img not found at URL http://kindofnormal.com/wumo.
```
But I can't reproduce the regex error

**Comment by TobiX** [comment]
Fixed in 2.16
